Tools/Dev/Design/README.md

```diff
@@ -60,21 +59,33 @@ Accent #FCBA19
 Links #1A5A96
 Navigation Bar #38598A
 Secondary Text #606060
-Background #F2F2F2
+Background #F7F7F7
 ```
 
-Or color live examples
-
 ### Buttons
 
-Describe the buttons
-
-<img src="images/buttons.png" alt="Buttons" width="100%" />
+Buttons are categorized and used as follow:
+Primary
+
+<img src="images/buttons/primary.png" alt="Primary Button" />
+<img src="images/buttons/secondary.png" alt="Secondary Button"  />
+<img src="images/buttons/highlighted.png" alt="Highlighted Button"  />
+<img src="images/buttons/navigation.png" alt="Navigation Button" />
+<img src="images/buttons/current nav.png" alt="Current Nav Button"/>
+<img src="images/buttons/link.png" alt="Secondary Button" />
+<img src="images/buttons/error link.png" alt="Primary Button"/>
+<img src="images/buttons/icon light.png" alt="Secondary Button" />
+<img src="images/buttons/icon dark.png" alt="Secondary Button" />
+<img src="images/buttons/submenu with icon.png" alt="Secondary Button" />
+<img src="images/buttons/submenu.png" alt="Submenu Button" />
```

Comment:
These images are very large, so they're hard to read in the document.  I suggest adding `width="300"` to the `<img />` tags and removing the newlines so the images can appear in-line.